### ISSUE TITLE
Issue forcing particle to render always on top of all scene objects 

### ISSUE THREAD

**Issue by johnnyrainbow** [Alchemist0823/three.quarks#84]
I want my particle to render on top over all other objects in the scene. however setting depthTest false on the material is not making it render on top as i'd expect, is there another way I should be approaching this? Thanks again, this framework is amazing!
```
 new ParticleSystem({ 
 ...
  material: new MeshBasicMaterial({
                map: this.texture,
                blending: AdditiveBlending,
                transparent: true,
                side: DoubleSide,
                
                //Here
                depthTest: false, 
             
            }),
            }
            ```

**Comment by Alchemist0823** [comment]
Thanks for your feedback I will add this option.

**Comment by Alchemist0823** [comment]
included in 47856a7572995eda560e14cc618022f2e5698750

**Comment by johnnyrainbow** [comment]
Awesome thanks 👍🤩
